### DISCUSSION TITLE
I looking for an Android client

### DISCUSSION THREAD

**Discussion by serrq** [DNSCrypt/dnscrypt-proxy#2328] [Q&A]
Does it exist a dnscrypt Android client available in f-droid? If yes, is it maintained?

**Comment by steinland1994** [comment]
Despite its name sounding like it is going to steal your bank account any second InViZiBle Pro is quite nice:

https://www.f-droid.org/en/packages/pan.alexander.tordnscrypt.stable/

https://github.com/Gedsh/InviZible

**Comment by karolyi** [comment]
I run Invizible + Trackercontrol, which I find an effective combo. Setting them both up takes a little patience, but totally worth it. No root needed, Trackercontrol acts as a VPN, and can be set up to use Invizible as a dnscrypt provider.
